src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/visitbalances/VisitBalanceResource.kt

```diff
@@ -242,15 +242,15 @@ data class PrisonerVisitBalanceResponse(
 @JsonInclude(JsonInclude.Include.NON_NULL)
 data class VisitBalanceAdjustmentResponse(
   @Schema(description = "Number of visit orders affected by the adjustment")
-  val remainingVisitOrders: Int? = null,
+  val visitOrderChange: Int? = null,
   @Schema(description = "Previous number of visit orders before the adjustment")
-  val previousRemainingVisitOrders: Int? = null,
+  val previousVisitOrderCount: Int? = null,
   @Schema(description = "Number of privileged visit orders affected by the adjustment")
-  val remainingPrivilegedVisitOrders: Int? = null,
+  val privilegedVisitOrderChange: Int? = null,
   @Schema(description = "Previous number of privileged visit orders before the adjustment")
-  val previousRemainingPrivilegedVisitOrders: Int? = null,
+  val previousPrivilegedVisitOrderCount: Int? = null,
   @Schema(description = "Adjustment reason")
-  val adjustmentReason: CodeDescription? = null,
```

Comment:
is this never null in the database then?

Reply:
nope - the column is nullable but there aren't any null values